docs/markdown/SimpleStart.md

```diff
@@ -91,6 +91,11 @@ versions.
 
 ### macOS
 
+#### With Homebrew
+`brew install meson ninja`
+
+Check out the [Homebrew project!](https://brew.sh)
```

Comment:
This line is not necessary.

Also missing a new subheader, to differentiate the official dmg installer from homebrew.